document/core/appendix/embedding.rst

```diff
@@ -43,6 +43,19 @@ In addition to the error conditions specified explicitly in this section, implem
    Implementations can refine it to carry suitable classifications and diagnostic messages.
 
 
+.. _embed-exception:
+
+Exceptions
+~~~~~~~~~~
+
+Invoke operations may throw or propagate exceptions, indicated by an auxiliary syntactic class:
+
+.. math::
+   \begin{array}{llllll}
+   \production{exception} & \exception &::=& \ETHROW & exnaddr & val^\ast \\
+   \end{array}
+
```

Comment:
Do we need to add language here describing what the tag address and payload values mean or is just naming them like this sufficient?